<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XYControllerW</class>
 <widget class="QMainWindow" name="XYControllerW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XY Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="PixmapDial" name="dial_x">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="PixmapDial" name="dial_y">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_x_controls">
        <property name="text">
         <string>X Controls:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_control_x"/>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_y_controls">
        <property name="text">
         <string>Y Controls:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_control_y"/>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_smooth">
        <property name="text">
         <string>Smooth</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>6</y>
         <width>864</width>
         <height>64</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>864</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>864</width>
         <height>64</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="PixmapKeyboard" name="keyboard" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>864</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>864</width>
            <height>64</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="act_quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="act_about"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <widget class="QMenu" name="menu_Channels">
     <property name="title">
      <string>Channels</string>
     </property>
     <addaction name="act_ch_01"/>
     <addaction name="act_ch_02"/>
     <addaction name="act_ch_03"/>
     <addaction name="act_ch_04"/>
     <addaction name="act_ch_05"/>
     <addaction name="act_ch_06"/>
     <addaction name="act_ch_07"/>
     <addaction name="act_ch_08"/>
     <addaction name="act_ch_09"/>
     <addaction name="act_ch_10"/>
     <addaction name="act_ch_11"/>
     <addaction name="act_ch_12"/>
     <addaction name="act_ch_13"/>
     <addaction name="act_ch_14"/>
     <addaction name="act_ch_15"/>
     <addaction name="act_ch_16"/>
     <addaction name="separator"/>
     <addaction name="act_ch_all"/>
     <addaction name="act_ch_none"/>
    </widget>
    <addaction name="menu_Channels"/>
    <addaction name="act_show_keyboard"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="act_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="act_show_keyboard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show MIDI &amp;Keyboard</string>
   </property>
  </action>
  <action name="act_ch_all">
   <property name="text">
    <string>(All)</string>
   </property>
  </action>
  <action name="act_ch_01">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="act_ch_02">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="act_ch_03">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="act_ch_04">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="act_ch_05">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="act_ch_06">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="act_ch_07">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="act_ch_08">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="act_ch_09">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="act_ch_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="act_ch_11">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </action>
  <action name="act_ch_12">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="act_ch_13">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </action>
  <action name="act_ch_14">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </action>
  <action name="act_ch_15">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </action>
  <action name="act_ch_16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </action>
  <action name="act_new">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="act_open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="act_save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="act_save_as">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="act_quit">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/16x16/application-exit.png</normaloff>:/16x16/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="act_ch_none">
   <property name="text">
    <string>(None)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PixmapKeyboard</class>
   <extends>QWidget</extends>
   <header>pixmapkeyboard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PixmapDial</class>
   <extends>QDial</extends>
   <header>pixmapdial.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>act_quit</sender>
   <signal>triggered()</signal>
   <receiver>XYControllerW</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
